taskfile/cache.go

```diff
@@ -33,6 +60,16 @@ func (c *Cache) write(node Node, b []byte) error {
 }
 
 func (c *Cache) read(node Node) ([]byte, error) {
+	fi, err := os.Stat(c.cacheFilePath(node))
```

Comment:
Maybe your could store the cacheFilePath in a variable, so you don't compute it twice in this method

Reply:
8e45a612c1a92ed43962400ec9192e77be0c370b will resolve.